src/uint/div.rs

```diff
@@ -159,6 +159,19 @@ impl<const LIMBS: usize> Uint<LIMBS> {
 
     /// Computes `self` % 2^k. Faster than reduce since its a power of 2.
     /// Limited to 2^16-1 since Uint doesn't support higher.
+    /// TODO: this is not constant-time.
```

Comment:
Sidebar: we should perhaps consider renaming this `rem2k_vartime` /cc @fjarri

Reply:
I imagine making it constant-time would also be pretty straightforward.

Although I am not sure we need it at all in the API.